### ISSUE TITLE
customArgs issue

### ISSUE THREAD

**Issue by zam6ak** [stevewillcock/grunt-msbuild#37]
In line 157 when adding customArgs, only array indexes are being added...

```
for (var customArg in options.customArgs) {
  args.push(customArg);
}
```

I think it should be 

```
for (var customArg in options.customArgs) {
  args.push(options.customArgs[customArg]);
}
```

**Comment by rhpijnacker** [comment]
Or, better yet:
`  options.customArgs.forEach(function(customArg) {
    args.push(customArg);
  }`

**Comment by stevewillcock** [comment]
Published as version 0.3.8 - thanks for the issue and code :)
